### DISCUSSION TITLE
Missing alignment jig - what to do?

### DISCUSSION THREAD

**Discussion by ghost** [thehighnibble/thehighnibble.github.io#163] [IMSAI8080esp replica]
So after some years I am finally building my IMSAI8080 kit. This is an older kit with the blue plastic case (mine was perfect) and the board shows "V3.4.2 July/2019". I've successfully followed the build instructions and videos to the point I've soldered the surface-mount RAM chip and the SD card holder, and completed the initial tests with the ESP32-PICO and SD card with success.

I'm now at the point where I start soldering in the LEDs and here found a problem. My kit has TWO of the slotted plastic pieces that raise the LEDs up off the board, but NO plastic piece with the holes for LED alignment.

I'm in Canada so even mailing an alignment jig today won't see it here inside a week or two.

Any suggestions? How best to align the LEDs without the jig?

Thanks.

**Comment by nbreeden2** [comment]
If you press the LEDs up tight against the slotted piece, I believe you can get the same results without the piece with the holes. I personally have always found the piece with the holes more difficult during soldering to keep in place. This of course may not be the same experience other have had.

With the LEDs held firmly to the slotted piece solder one leg of the LED. Once all eight LEDs in a section have one lead soldered come back and one led at a time press the LED firmly and reflow the solder on it's one soldered pin. With the bottom of the body of the LED flush with the slotted piece the LEDs fall into alignment. 

I'd start by soldering one LED on each end of the slotted piece and get it soldered in place (both leads soldered and the LED aligned) then you can insert the remaining LEDs one at a time. There will be enough friction from the two end LEDs to hold the slotted piece in place.

A piece of blue painter tape in the middle of the slotted piece will help hold it in place while you solder the end LEDs.

For each set I've soldered the slotted piece is always a bit difficult to remove due to the friction from the soldered LEDs.

From experience I know a small amount of misalignment won't be visible with the kit assembled.

I've built several of the kits and in the end have only used the piece with the holes after the LEDs are soldered as I always misalign one or more through handling as I solder the resistors etc. You can of course straighten them back visually.

-Neil

**Comment by ghost** [comment]
Neil,
That's exactly what I did. I've built enough kits now that I'm confident in the "solder one and adjust" technique to be happy with it. I'm actually now at the stage where I have the switch black panel in place and am about to install the switches. Everything checked out perfectly in tests so far. I even managed to solder the two surface mount components on my first try. In truth, I had more difficulty with that transistor hole spacing than with the surface mount components. I should have used my flux pen but I managed to get it correct (no solder bridges) in the end.

The only place where I might use the plans to build a LED alignment tool is when you invariably bend an LED during fiddly component installation. I bent two very slightly while installing the ICs. I was able to realign, but there's a place the hole guile would be nice.
